release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,40 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	wget --quiet "$artifactsUrl"
```

Comment:
We should probably be a bit more secure here and verify that the build server is actually our one with a TLS certificate check. Not just blindly trusting **all** the global CAs.

```suggestion
	wget --quiet --ca-cert=/etc/ssl/certs/ISRG_Root_X1.pem "$artifactsUrl"
```

Reply:
works for me, although it means that this automation will break if ever we decide to rely on something which isn't signed by that CA